### PR TITLE
Updated the docs for GitHub Releases

### PR DESCRIPTION
**Pull request by cheesesashimi**

### CODE REVIEW

**Comment by BanzaiMan** [comment]
See also: https://github.com/travis-ci/dpl/issues/344.

**Comment by cheesesashimi** [comment]
@BanzaiMan: If you'd like me to hold off on the Octokit env variable portion of this PR, I'm happy to do so until your PR is more fleshed out.

**Comment by BanzaiMan** [comment]
@cheesesashimi No worries. I'm working with @danishkhan to test both of these approaches at the moment.

Thanks for the ping.

File: user/deployment/releases.md [hunk 1]
```diff
@@ -58,6 +66,17 @@ You can upload multiple files using yml array notation. This example uploads two
         tags: true
 
 
+You can also enable wildcards by setting `file_glob` to `true`. This example will include all files in a given directory.
```
Comment (niccokunzmann) [inline user/deployment/releases.md]: This is so useful! I want it on the site! 👍

**Comment by plaindocs** [comment]
Thanks!

**Pull request merged**